frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -192,7 +192,7 @@
               }
 
               .device_mapping__cell,
-              .mdm_enrollment_status__cell,
+              .mdm_enrollment_status__cell, // TODO: not sure if this should be modified...
```

Comment:
I think this will work as-is with the optional id approach that I outlined in `HostTableConfig` above.